src/index.html

```diff
@@ -49,16 +49,135 @@
+                    <svg class="hero__link_inner" width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M22 40.3333C32.1252 40.3333 40.3333 32.1252 40.3333 22C40.3333 11.8747 32.1252 3.66663 22 3.66663C11.8747 3.66663 3.66663 11.8747 3.66663 22C3.66663 32.1252 11.8747 40.3333 22 40.3333Z" stroke="#61BED9" stroke-linecap="round" stroke-linejoin="round"/>
+                        <path d="M14.6666 22L22 29.3333L29.3333 22" stroke="#61BED9" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                        <path d="M22 14.6666V29.3333" stroke="#61BED9" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                    </svg>
+                    <a class="hero__link_text" href="#">Read More</a>
+                </div>
+            </div>
+            <div class="hero__right">
+                <img class="hero__images hero__images_top" src="./images/title_house.png" alt="" width="460" height="460">
+                <img class="hero__images hero__images_bottom" src="./images/title_family.png" alt="" width="252" height="252">
+                <img class="hero__images hero__images_dots" src="./images/dots.png" alt="" width="372" height="320">     
+            </div>
         </div>
     </section>
-    <section></section>
-    <section></section>
-    <section></section>
+    <section class="mosaic">
+        <div class="mosaic__vertical-text">
+            <p>Mosaic</p>
+        </div>
+        <div class="container mosaic__container">
+            <div class="mosaic__backgroud-color">
+                <h2 class="mosaic__title">Mosaic Management Group</h2>
+                <p class="mosaic__text">In order for an HOA to effectively upkeep and manage the responsibilities required of their board, 
+                    expert third party help is a necessity. Mosaic Management Group is an HOA property management firm that brings together 
+                    all of the expertise and resources a community needs for effective management of its grounds and tenants.</p>
+                <p class="mosaic__text">We understand every community is unique. That’s why we focus on planting our roots within your
+                    neighborhood, allowing us to provide essential duties that help a board navigate the challenges and changes that come with HOA 
+                    management.</p>
+                <p class="mosaic__text">Trust our organizations to help your HOA board manage its community with professionalism and quality 
+                    customer service.</p>
+            </div>
+            <img class="mosaic__img" src="./images/mosaic_foto.png" alt="">
+        </div>
+    </section>
+    <section class="management">
+        <div class="container management__container">
+            <div class="management__left">
+                <img class="management_image" src="./images/management-image.png" alt="" width="455" height="534">
```

Comment:
Please, use css background image for this type of images, because this is not a content image

```
<span style="background-image: url(./images/management-image.png)"></span>
```